src/services/address_service.js

```diff
@@ -0,0 +1,78 @@
+import baseService from './base_service';
+
+export default (easypostClient) =>
+  class AddressService extends baseService(easypostClient) {
+    static _name = 'Address';
+
+    static _url = 'addresses';
+
+    static key = 'address';
+
+    /**
+     * Create an address.
+     * @param {*} params
+     * @returns {Address}
+     */
+    static async create(params) {
```

Comment:
When users access the address service via `myClient.Address`, they are accessing an instance of this `AddressService` class. All API functions in this class (and all service classes) should be instance-based, not static.

Reply:
hmmm... indeed. This'll take a bit of work.

Reply:
Per internal conversations, it seems like this is actually the correct way to do it because Node is magic.